CHANGELOG.md

```diff
@@ -24,6 +25,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Allow form messages to go on multiple lines
 - On the search page, improve accessibility of the filters pane when using a
   small screen
+- Rename docker `node` service to `frontend` 
```

Comment:
It concerns only configuration for dev purpose, no need to add an entry in the changelog about that.